.github/stale.yml

```diff
@@ -8,6 +8,7 @@ exemptLabels:
   - "State: demonstrator"
   - "State: don't stale"
   - "Type: tracking"
+  - "Type: bug"
```

Comment:
Please keep the alphabetic order though.

Reply:
> Please keep the alphabetic order though.

Done @miri64